tests/API/axios.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test } from '@playwright/test';
+import axios, { Axios } from 'axios';
+import { wrapper } from 'axios-cookiejar-support';
+import { CookieJar } from 'tough-cookie';
+
+test.describe(async () => {
+    let client: Axios;
+    let userId: string;
+    let token: string;
+
+    test.beforeAll(async () => {
+        const jar = new CookieJar();
+        client = wrapper(axios.create({
+            baseURL: 'https://demoqa.com/',
+            validateStatus: function (status) {
+                return status >= 200 && status < 300
+            },
+            jar
+        }));
+        const loginData = await client.post('Account/v1/Login', {
+            userName: "testUserArtemQwerty",
+            password: "Qwe12345!"
+        });
+        userId = loginData.data.userId;
+        token = loginData.data.token;
+        console.log('token -------________--------________', token)
+        console.log(loginData.status);
+        console.log(loginData.data);
+        console.log(loginData.headers);
+        client.defaults.headers.common['Authorization'] = `Bearer ${token}`;
+    });
+
```

Comment:
you can add delete all books, before 
```suggestion
    test('Delete all books before', async () => {
        let actualUserBooks = await client.get(`/Account/v1/User/${userId}`)
        console.log(actualUserBooks.data);
        if (await actualUserBooks.data.books.length > 0) {
            for (let index = 0; index < await await actualUserBooks.data.books.length; index++) {
                const deleteBookResponse = await client.delete('/BookStore/v1/Book', {
                    data: {
                        "isbn": actualUserBooks.data.books[0].isbn,
                        "userId": userId
                    }
                });
                expect(deleteBookResponse.status).toBe(204);
            }
        }
    })
```